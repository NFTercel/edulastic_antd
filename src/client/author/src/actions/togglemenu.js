import { TOGGLE_MENU } from "../constants/actions";

export const toggleSideBarAction = () => ({
  type: TOGGLE_MENU
});
