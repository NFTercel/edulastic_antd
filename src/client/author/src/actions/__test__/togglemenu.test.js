import { desktopSideBar } from "../togglemenu";

describe("Toggle Menu", () => {
  it("desktop sidebar should return an action", () => {
    expect(desktopSideBar()).toMatchSnapshot();
  });
});
