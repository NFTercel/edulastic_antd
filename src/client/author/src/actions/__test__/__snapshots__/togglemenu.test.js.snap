// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Toggle Menu desktop sidebar should return an action 1`] = `
Object {
  "type": "[homeUI] toggle menu",
}
`;
